tests/behat/bootstrap/FeatureContext.php

```diff
@@ -522,4 +522,19 @@ public function wysiwygFillField(string $field, string $value): void {
         ");
   }
 
+  /**
+   * @Then /^I should see the text "([^"]*)" in the same row as "([^"]*)"$/
+   */
+  public function iShouldSeeTheTextInTheSameRowAs($text1, $text2)
```

Comment:
DrupalContext already provides  `@Then I should see (the text ):text in the :rowText row`

`assertTextInTableRow`

Reply:
Updated.